ts/components/conversation/media-gallery/DocumentListItem.tsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * @prettier
+ */
+import React from 'react';
+
+import moment from 'moment';
+import formatFileSize from 'filesize';
+
+interface Props {
+  fileName?: string;
+  fileSize?: number;
+  i18n: (key: string, values?: Array<string>) => string;
+  onClick?: () => void;
+  timestamp: number;
+}
+
+const styles = {
+  container: {
+    width: '100%',
+    height: 72,
+    borderBottomWidth: 1,
+    borderBottomColor: '#ccc',
+    borderBottomStyle: 'solid',
+  },
+  itemContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    flexWrap: 'nowrap',
+    alignItems: 'center',
+    height: '100%',
+  } as React.CSSProperties,
+  itemMetadata: {
+    display: 'inline-flex',
+    flexDirection: 'column',
+    flexGrow: 1,
+    flexShrink: 0,
+    marginLeft: 8,
+    marginRight: 8,
+  } as React.CSSProperties,
+  itemDate: {
+    display: 'inline-block',
+    flexShrink: 0,
+  },
+  itemIcon: {
+    flexShrink: 0,
+  },
+  itemFileSize: {
+    display: 'inline-block',
+    marginTop: 8,
+    fontSize: '80%',
+  },
+};
+
+export class DocumentListItem extends React.Component<Props, {}> {
+  public renderContent() {
+    const { fileName, fileSize, timestamp } = this.props;
+
+    return (
+      <div style={styles.itemContainer} onClick={this.props.onClick}>
+        <img
+          src="images/file.svg"
+          width="48"
+          height="48"
+          style={styles.itemIcon}
+        />
+        <div style={styles.itemMetadata}>
+          <strong>{fileName}</strong>
```

Comment:
Do we want to use tags like this, or rely on CSS only? I don't think we use `<strong>` or `<em>` anywhere else in the codebase.

Reply:
Good point. I could do `<span style={styles.fileName}>` or similar 👍